### PR TITLE
Update act-rules-format.bs

### PR DESCRIPTION
**Pull request by kengdoj**
Add required Rules Format Version

### CODE REVIEW

File: act-rules-format/act-rules-format.bs [hunk 1]
```diff
@@ -589,6 +590,11 @@ It is important to keep track of versions of ACT Rules so that users of the rule
 This section was previously named "Change Log". Either section name is acceptable.
 
 
+Rules Format Version {#rules-format-version}
```
Comment (daniel-montalvo) [inline act-rules-format/act-rules-format.bs:593]:
```suggestion
ACT Rules Format Version {#act-rules-format-version}
```

To be consistent with spec name

File: act-rules-format/act-rules-format.bs [hunk 2]
```diff
@@ -589,6 +590,11 @@ It is important to keep track of versions of ACT Rules so that users of the rule
...
+-------------------------------------------
+
+ACT Rules <em class="rfc2119">must</em> indicate the rule's compatibility to a version of the Rules Format as each version has different requirements.
```
Comment (daniel-montalvo) [inline act-rules-format/act-rules-format.bs:596]:
```suggestion
ACT Rules <em class="rfc2119">must</em> indicate the rule's compatibility to a version of the ACT Rules Format as each version has different requirements. 
```

Same as above.

File: act-rules-format/act-rules-format.bs [hunk 3]
```diff
@@ -1015,9 +1021,10 @@ This section is *non-normative*.
 
 <ul>
   <li><strong>4. Rule Structure</strong> Accessibility Support and Assumptions section are now subsections of Background.</li>
-  <li><strong>4.4. Accessibility Requirements Mapping</strong> Accessibility requirements are now categorized as either Conformance requirements or Secondary requirements.<li>
+  <li><strong>4.4. Accessibility Requirements Mapping</strong> Accessibility requirements are now categorized as either Conformance requirements or Secondary requirements.</li>
   <li><strong>4.8. Background</strong> New optional Related Rules and Other Resources subsections have been added</li>
   <li><strong>4.11. Change Log</strong> The Change Log has been renamed to Rule Versions</li>
+  <li><strong>4.12 Rules Format Version</strong> New requirement to identify rules format version compatiblity</li>
```
Comment (daniel-montalvo) [inline act-rules-format/act-rules-format.bs:1027]:
```suggestion
  <li><strong>4.12 ACT Rules Format Version</strong> New requirement to identify ACT Rules Format version compatibility</li>
```

**Review by daniel-montalvo — CHANGES_REQUESTED** [review]
I'd suggest to use the spec name "ACT Rules Format" Not feeling strongly though. throughout. I also caught a typo.

File: act-rules-format/act-rules-format.bs [hunk 4]
```diff
@@ -106,6 +106,7 @@ An ACT Rule <em class="rfc2119">must</em> consist of at least the following item
     * [Other Resources (optional)](#other-resources)
 * [Test Cases](#test-cases)
 * [Rule Versions](#rule-versions)
+* [Rules Format Version](#rules-format-version)
```
Comment (kengdoj) [inline act-rules-format/act-rules-format.bs:109]:
```suggestion
* [ACT Rules Format Version](#act-rules-format-version)
```

File: act-rules-format/act-rules-format.bs [hunk 5]
```diff
@@ -589,6 +590,11 @@ It is important to keep track of versions of ACT Rules so that users of the rule
 This section was previously named "Change Log". Either section name is acceptable.
 
 
+ACT Rules Format Version {#act-rules-format-version}
+-------------------------------------------
+
+ACT Rules <em class="rfc2119">must</em> indicate the rule's compatibility to a version of the ACT Rules Format as each version has different requirements.
```
Comment (WilcoFiers) [inline act-rules-format/act-rules-format.bs:596]: > An ACT Rule is written for a specific version of the ACT Rules Format. Each rule MUST indicate which version of the rules format it is written for.

Comment (kengdoj) [inline act-rules-format/act-rules-format.bs:596]:
```suggestion
An ACT Rule is written for a specific version of the ACT Rules Format. Each ACT Rule <em class="rfc2119">MUST</em> indicate the version of the ACT Rules Format for which it is written. 
```

**Review by daniel-montalvo — APPROVED** [review]

File: act-rules-format/act-rules-format.bs [hunk 6]
```diff
@@ -589,6 +590,11 @@ It is important to keep track of versions of ACT Rules so that users of the rule
 This section was previously named "Change Log". Either section name is acceptable.
 
 
+ACT Rules Format Version {#act-rules-format-version}
+-------------------------------------------
+
+An ACT Rule is written for a specific version of the ACT Rules Format. Each ACT Rule <em class="rfc2119">MUST</em> indicate the version of the ACT Rules Format for which it is written.
```
Comment (kengdoj) [inline act-rules-format/act-rules-format.bs:596]:
```suggestion
An ACT Rule is written for a specific version of the ACT Rules Format. Each ACT Rule <em class="rfc2119">must</em> indicate the version of the ACT Rules Format for which it is written. 
```

**Review by WilcoFiers — APPROVED** [review]

**Review by tbostic32 — APPROVED** [review]

File: act-rules-format/act-rules-format.bs [hunk 7]
```diff
@@ -1015,9 +1021,10 @@ This section is *non-normative*.
 
 <ul>
   <li><strong>4. Rule Structure</strong> Accessibility Support and Assumptions section are now subsections of Background.</li>
-  <li><strong>4.4. Accessibility Requirements Mapping</strong> Accessibility requirements are now categorized as either Conformance requirements or Secondary requirements.<li>
+  <li><strong>4.4. Accessibility Requirements Mapping</strong> Accessibility requirements are now categorized as either Conformance requirements or Secondary requirements.</li>
   <li><strong>4.8. Background</strong> New optional Related Rules and Other Resources subsections have been added</li>
   <li><strong>4.11. Change Log</strong> The Change Log has been renamed to Rule Versions</li>
+  <li><strong>4.12 ACT Rules Format Version</strong> New requirement to identify ACT Rules Format version compatibility</li>
   <li><strong>4.14. Implementations<strong> New section has been added, including a method for determining consistency with ACT Rules</li>
```
Comment (WilcoFiers) [inline act-rules-format/act-rules-format.bs:1028]:
```suggestion
  <li><strong>4.15. Implementations<strong> New section has been added, including a method for determining consistency with ACT Rules</li>
```

**Review by tombrunet — APPROVED** [review]

**Pull request merged**